src/main/java/org/spongepowered/common/config/category/WorldCategory.java

```diff
@@ -103,6 +103,15 @@
 
     @Setting(value = "weather-ice-and-snow", comment = "Enable to allow the natural formation of ice and snow in supported biomes.")
     private boolean weatherIceAndSnow = true;
+
+    public static final int USE_SERVER_VIEW_DISTANCE = -1;
```

Comment:
Should this be exposed in the API?

Reply:
Would a `clearCustomViewDistance()` method make sense instead?